core/main/src/firebolt/firebolt_ws.rs

```diff
@@ -221,9 +221,6 @@ impl FireboltWs {
             return;
         }
         if !gateway_secure {
-            state
-                .session_state
-                .add_pending_session(app_id.clone(), None);
```

Comment:
Unrelated but pending sessions are added when perms are checked upon app launch, not needed here.